configure.ac

```diff
@@ -1924,7 +1924,9 @@ dnl ---------------
 dnl gRPC
 dnl ---------------
 if test "$enable_grpc" = "yes"; then
-  PKG_CHECK_MODULES([GRPC], [grpc grpc++ protobuf], [
+  AC_LANG([C++])
```

Comment:
This changes the language for the entire remainder of `configure.ac`, can you fix it to use `AC_LANG_PUSH` + `AC_LANG_POP` afterwards?

Reply:
I pushed the change, but I don't know if it's picking it up since this is merged now. SHould I do another PR?